README.md

```diff
@@ -118,7 +118,7 @@ instances to replicate the state. A synchronization library should also
 be developed for this milestone.
 
 - [x] Migrate to a fully async interface
-- [ ] :runner: Pluggable encoding for nodes, for optional compatibility with MerkleDB
+- [x] Pluggable encoding for nodes, for optional compatibility with MerkleDB
```

Comment:
I don't think hash compatibility has been tested has it?

Reply:
Yeah, that should be covered by `MerkleDB root hash in parity for seamless transition between MerkleDB and Firewood.`